--- origsrc/lua-gd-2.0.33r2/Makefile	2006-05-03 20:03:48.000000000 -0500
+++ src/lua-gd-2.0.33r2/Makefile	2008-02-10 22:34:03.375000000 -0600
@@ -34,7 +34,7 @@
 
 # Name of .pc file. "lua5.1" on Debian/Ubuntu
 LUAPKG=lua5.1
-OUTFILE=gd.so
+OUTFILE=gd.dll
 CFLAGS=`gdlib-config --cflags` `pkg-config $(LUAPKG) --cflags` -O3 -Wall
 GDFEATURES=`gdlib-config --features |sed -e "s/GD_/-DGD_/g"`
 LFLAGS=-shared `gdlib-config --ldflags` `gdlib-config --libs` \
@@ -67,11 +67,12 @@
 all: $(OUTFILE)
 
 $(OUTFILE): luagd.c
-	$(CC) -o $(OUTFILE) $(GDFEATURES) $(CFLAGS) $(LFLAGS) luagd.c
+	$(CC) -o $(OUTFILE) $(GDFEATURES) $(CFLAGS) luagd.c $(LFLAGS)
 	lua test_features.lua
 
 install: $(OUTFILE)
-	install -s $(OUTFILE) $(INSTALL_PATH)
+	mkdir -p $(DESTDIR)$(INSTALL_PATH)
+	install -s $(OUTFILE) $(DESTDIR)$(INSTALL_PATH)
 
 clean:
 	rm -f $(OUTFILE) *.o
